src/main/java/Bonnie/Bonnie.java

```diff
@@ -0,0 +1,19 @@
+package Bonnie;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Bonnie {
+
+    private static String username;
+    private static ArrayList<Task> tasklist = new ArrayList<>();
+
+    public static void main(String[] args) throws EmptyTodoException, UnknownCommandException, DeadlineFormatException, IOException {
+        startApp();
+    }
+
+    protected static void startApp() throws DeadlineFormatException, EmptyTodoException, UnknownCommandException {
+        Storage.init();
+        Ui.init();
```

Comment:
```suggestion
        Storage.init();
        Ui.init();
```

no suggestions, I personally didn't think of initializing my Storage and UI like this. 
This looks cleaner :)